x-pack/examples/triggers_actions_ui_example/public/components/rules_list_sandbox.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { TriggersAndActionsUIPublicPluginStart } from '@kbn/triggers-actions-ui-plugin/public';
+
+interface SandboxProps {
+  triggersActionsUi: TriggersAndActionsUIPublicPluginStart;
+}
+
+const style = {
+  flex: 1,
+};
+
+export const RulesListSandbox = ({ triggersActionsUi }: SandboxProps) => {
+  return <div style={style}>{triggersActionsUi.getRulesList({})}</div>;
```

Comment:
This is really cool, but I'll be honest - I'm surprised that the `getRulesList` wasn't just the RulesList, but rather the whole experience - the rules list, with generic creation and editing flyout.

As a result the "Rules List' page actually loads as the whole Stack Management rules tab - which creates rules in the main rules view, not just in the example plugin. This behaviour surprised me as a user.

I **don't** think this is a blocker to merging this PR.
But I do wonder if we should rename this to something less specific to the rules list?
And I wonder if we should _also_ expose the RulesList as a list that relies on hard-coded mock rules (presumably creation would be disabled?) and that way developers can see how the rules list itself can be used in as a contained sharable component?

This can be a follow-up PR and doesn't necessarily have to be done in the short term IMHO.

<img width="1290" alt="Screenshot 2022-07-19 at 09 35 29" src="https://user-images.githubusercontent.com/386208/179709759-24be4bc5-dd7b-4867-adc1-336da1d8f51a.png">

Reply:
haha I'd say both 😉 

Like I said - this is not a blocker!
I'd consider having two pages. -one for the list with mock data, and one for the whole injectable component that we can run tests against.

Reply:
We can totally do that in a separate PR though, not a blocker:)